R/create.R

```diff
@@ -184,41 +175,41 @@ create_from_github <- function(repo_spec,
     ssh = repo_info$ssh_url
   )
 
-  done("Cloning repo from ", value(origin_url), " into ", value(repo_path))
+  done(glue("Cloning repo from {value(origin_url)} into {value(repo_path)}"))
```

Comment:
I wonder if we should just redefine done, todo etc. to use glue directly rather than paste0?

https://github.com/r-lib/usethis/blob/ca21abeb936ff75d41a3f39e421a919d7ba1538c/R/style.R#L11-L13

e.g.

```r
done <- function(...) {
  bullet(glue(...), bullet = crayon::green(clisymbols::symbol$tick))
}